mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/store/MantisStorageProvider.java

```diff
@@ -16,171 +16,86 @@
 
 package io.mantisrx.server.master.store;
 
-import io.mantisrx.master.resourcecluster.DisableTaskExecutorsRequest;
-import io.mantisrx.server.master.resourcecluster.ClusterID;
-import io.mantisrx.server.master.resourcecluster.TaskExecutorID;
-import io.mantisrx.server.master.resourcecluster.TaskExecutorRegistration;
 import java.io.IOException;
 import java.util.List;
-import rx.Observable;
+import java.util.Map;
 
 
 public interface MantisStorageProvider {
```

Comment:
A couple of questions:
1. Do we need this abstraction since this looks like it assumes that the underlying data will get stored in a KV store? 
2. If we need it, should we rename this to something like KVStore or something like that?

Reply:
done, good suggestion.

Reply:
so are you deleting this class?